app/containers/MessageBox/forceJpgExtension.ts

```diff
@@ -0,0 +1,12 @@
+import { ImageOrVideo } from 'react-native-image-crop-picker';
+
+import { isIOS } from '../../utils/deviceInfo';
+
+const regex = new RegExp(/\.[^/.]+$/); // Check from last '.' of the string
+
+export const forceJpgExtension = (attachment: ImageOrVideo): ImageOrVideo => {
+	if (isIOS && attachment.mime === 'image/jpeg' && attachment.filename) {
+		attachment.filename = attachment.filename.replace(regex, '.jpg'); // Replace files that ends with .jpg | .heic | .jpeg to .jpg;
```

Comment:
It's not `.jpg | .heic | .jpeg` only

Reply:
You are right, so I changed the comment to // Replace files extension that mime type is 'image/jpeg' to .jpg;

WDYT?